config/samples/nodeobservability_v1alpha1_nodeobservability.yaml

```diff
@@ -3,6 +3,6 @@ kind: NodeObservability
 metadata:
   name: cluster
 spec:
-  labels:
+  nodeSelector:
```

Comment:
Other sample files need to be updated too: `nodeobservability_v1alpha1_nodeobservability-all.yaml` and `nodeobservability_v1alpha1_nodeobservabilitymachineconfig.yaml`.